src/BitOasis/Bitfinex/ExchangeRate/RabbitMqExchangeRateUpdater.php

```diff
@@ -19,12 +19,16 @@
 class RabbitMqExchangeRateUpdater implements LoggerAwareInterface {
 	use LoggerAwareTrait;
 
-	const WEB_RPC_EXCHANGE = 'bitoasis.web-events';
+	const WEB_RPC_EXCHANGE = 'bitoasis.fiat-rates';
```

Comment:
Constant name is confusing. Same for other below

Reply:
Changed